packages/account/src/Sections/Security/Passwords/change-password-form.jsx

```diff
@@ -1,25 +1,26 @@
-import classNames from 'classnames';
 import React from 'react';
 import { Formik } from 'formik';
-import { FormSubmitErrorMessage, Button, Loading, PasswordInput, PasswordMeter } from '@deriv/components';
+import { Loading, PasswordInput, PasswordMeter } from '@deriv/components';
 import { withRouter } from 'react-router-dom';
 import { routes, isMobile, validPassword, validLength, getErrorMessages } from '@deriv/shared';
 import { localize } from '@deriv/translations';
 import { WS } from 'Services/ws-methods';
 import { connect } from 'Stores/connect';
-import FormSubHeader from 'Components/form-sub-header';
-import FormBody from 'Components/form-body';
-import FormFooter from 'Components/form-footer';
+import PasswordsFooter from './passwords-footer.jsx';
 
 class ChangePasswordForm extends React.Component {
```

Comment:
Can we refactor this component into functional? 🙏🏻

Reply:
Sure, I will look into it